src/connector.js

```diff
@@ -861,13 +966,14 @@ function updateResultListState( newState ) {
 				else {
 					author = result.raw.author;
 				}
-				if( params.isContextSearch ) {
-					author = author.replace( ';', ', ' );
-				}
-				else {
-					author = author.replace( ',', ';' );
-					author = author.replace( ';' , '</li> <li>' );
-				}
+
+				author = author.replaceAll( ';' , '</li> <li>' );
+			}
+
+			let breadcrumb = "";
+			if ( result.raw.hostname && result.raw.displaynavlabel ) {
+				const splittedNavLabel = ( Array.isArray( result.raw.displaynavlabel ) ? result.raw.displaynavlabel[0] : result.raw.displaynavlabel).split( '>' );
+				breadcrumb = result.raw.hostname + '&nbsp;</li><li>' + splittedNavLabel[splittedNavLabel.length-1];
```

Comment:
If there are no breadcrumb available, then it should fall back on the **full URL** (which is the case, but) with the following semantics:
```
<p class="location">
	<cite>
		<a href="https://www.canada.ca/en.html">https://www.canada.ca/en.html</a>
	</cite>
</p>
```

Reply:
As per our reference implementation: https://wet-boew.github.io/GCWeb/templates/search/results-en.html